Modules/Module_Processing/Module_Population/xASL_im_CreateAnalysisMask.m

```diff
@@ -113,11 +114,19 @@
 	%% B) Create, combine & save vascular, susceptibity & FoV masks
 	MaskVascular = xASL_io_Nifti2Im(PathVascularMask)>=Threshold;
     MaskFoV = xASL_io_Nifti2Im(PathFoV)>=Threshold;
-        
+    
+
+
     % Legacy Susceptibility Masking
-    [x,DoSusceptibility, TemplateMask, ThresholdSuscept, MaskSusceptibility] = ...
-    xASL_im_CreateAnalysisMask_LegacySusceptibilityMasking(x, PathSusceptibilityMask, Threshold, MaskFoV);
-                    
+    MaskSusceptibility = xASL_io_Nifti2Im(PathSusceptibilityMask);

```

Comment:
I do not understand. How can we load the mask that we will be creating only a few lines lower?

Reply:
Same as previous comment